packages/renderer/src/lib/onboarding/OnboardingEmbeddedComponent.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+import { onMount } from 'svelte';
+import { configurationProperties } from '/@/stores/configurationProperties';
+import { providerInfos } from '/@/stores/providers';
+import type { IConfigurationPropertyRecordedSchema } from '../../../../main/src/plugin/configuration-registry';
+import type { ProviderInfo } from '../../../../main/src/plugin/api/provider-info';
+import PreferencesConnectionCreationRendering from '../preferences/PreferencesConnectionCreationRendering.svelte';
+import type { OnboardingEmbeddedComponentId } from '../../../../main/src/plugin/api/onboarding';
+
+export let embeddedComponent: OnboardingEmbeddedComponentId;
+export let extensionId: string;
+
+let configurationItems: IConfigurationPropertyRecordedSchema[];
+$: configurationItems;
+let providers: ProviderInfo[];
+let providerInfo: ProviderInfo | undefined;
+$: providerInfo;
+
+onMount(() => {
+  configurationProperties.subscribe(value => {
+    configurationItems = value;
+  });
+
+  providerInfos.subscribe(value => {
+    providers = value;
+    providerInfo = providers.find(p => p.extensionId === extensionId);
+  });
+});
+</script>
+
+{#if embeddedComponent === 'create' && providerInfo && configurationItems}
```

Comment:
I think 100% we need to add documentation to this part as unlike the other micromark features this seems really hidden.

If a provider such as Red Hat Sandbox were to add `create` to their extension, they wouldn't be able to figure it out through package.json / as well as other "onboarding-related" documentation.

With this PR, we are requiring that the provider has a `containerProviderConnection` provided.

We also have a kubernetes-specific one as well? So it's two things were misssing documentation for.

I'm fine if we create an issue for this so we can make sure we document the update.

Reply:
Yes, I still didn't add any doc for the onboarding as i was waiting it to settle down a bit. Or i had to rewrite it from scratch almost every PR as it changed a lot from the first implementation. 
I should be close to have a working onboarding for podman (install -> set some setting -> create/start machine), then i'll work on https://github.com/containers/podman-desktop/issues/3633 which covers all aspects of how to create an onboarding (including the supported embedded components).

Regarding the kubernetes-specific one - not sure if i got what you're saying but the code support the `kubernetesProviderConnectionCreation`. So if we create a  workflow for `kind` it should work fine.